proposals/nnnn-actors.md

```diff
@@ -738,3 +723,27 @@ There are a few reasons why we are not currently comfortable including task-chai
 * The approach is semantically similar to recursive locks (even though there is no actual locking or blocking involved), which are particularly hard to work with in concurrent code, and we have not yet convinced ourselves those problems won't manifest in the actor model.
 
 If we can address the above, task-chain reentrancy can be introduced into the actor model with another spelling of the reentrancy attribute such as `@reentrant(task)`, and may provide a more suitable default than non-reentrant (`@reentrant(never)`).
+
+### Eliminating inheritance
+
+Like classes, actors as proposed allow inheritance. However, actors and classes cannot be co-mingled in an inheritance hierarchy, so there are essentially two different kinds of type hierarchies. It has been [proposed](TODO) that actors should not permit inheritance at all, because doing so would simplify actors: features such as method overriding, initializer inheritance, required and convenience initializers, and inheritance of protocol conformances would not need to be specified, and users would not need to consider them. The [discussion thread](TODO) on the proposal to eliminate inheritance provides several reasons to keep actor inheritance:
+
+* Actor inheritance makes it easier to port existing class hierarches to get the benefits of actors. Without actor inheritance, such porting will also have to content with (e.g.) replacing superclasses with protocols and explicitly-specified stored properties.
+* The lack of inheritance in actors won't prevent users from having to understand the complexities of inheritance, because inheritance will still be used pervasively with classes.
+* The design and implementation of actors naturally admits inheritance. Actors are fundamentally class-like, the semantics of inheritance follow directly from the need to maintain actor isolation, and the implementation of actors as "special classes" naturally supports all of these features. There is little benefit to the implementation from eliminating the possibility of inheritance of actors.
+
```

Comment:
That's incomplete / not concluded yet right?

Reply:
I'll add a conclusion, but yes, this is still a point of debate.

Reply:
Ok 👍